sites/skeleton.dev/src/routes/(inner)/elements/forms/+page.svelte

```diff
@@ -7,6 +7,18 @@
 	// Field Settings
 	import { fieldSettings } from './settings';
 
+	interface InputSettings {
+    title: string;
+    type: HTMLInputElement['type'];
+    placeholder?: string;
+    readonly?: boolean;
+    disabled?: boolean;
+    multiple?: boolean;
+    autocomplete?: string;
+    tabindex?: string;
```

Comment:
this should be `tabindex?: number;` instead.
the same change must be applied in `/elements/forms/settings.ts`

Reply:
If I implement this change, line 16 needs a change as well